src/test/kotlin/model/GenomeTest.kt

```diff
@@ -22,19 +22,19 @@ class GenomeTest : FunSpec({
   }
 
   test("take") {
-    GenMutator(Config.test).random().take(3) shouldBe listOf(Frp, NAC, Frp)
+    GenomeManager(Config.test).random().take(3) shouldBe listOf(Frp, NAC, Frp)
   }
 
   test("takeLast") {
-    GenMutator(Config.test).random().takeLast(3) shouldBe listOf(Frp, NAC, MDM2)
+    GenomeManager(Config.test).random().takeLast(3) shouldBe listOf(Frp, NAC, MDM2)
   }
 
   test("drop") {
-    GenMutator(Config.test).random().drop(3) shouldBe listOf(DmNotch, Frp, Frp, NAC, MDM2)
+    GenomeManager(Config.test).random().drop(3) shouldBe listOf(DmNotch, Frp, Frp, NAC, MDM2)
   }
 
   test("dropLast") {
-    GenMutator(Config.test).random().dropLast(3) shouldBe listOf(Frp, NAC, Frp, DmNotch, Frp)
+    GenomeManager(Config.test).random().dropLast(3) shouldBe listOf(Frp, NAC, Frp, DmNotch, Frp)
```

Comment:
are you going to change these tests and it's just a refactor or should I take issue with you using so much random?

Reply:
I clicked refactor